src/consumer.ts

```diff
@@ -1,6 +1,7 @@
+import { Logger } from "winston"
 import { Message } from "./producer"
 
-export type ConsumerFunc = (message: Message) => void
+export type ConsumerFunc = (message: Message, logger: Logger) => void
```

Comment:
Why have you added logger?

Reply:
I think it happened during the rebase, i will remove it